# Generated by Django 4.1.13 on 2024-02-02 08:35

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('user', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='customuser',
            name='certificate',
            field=models.FileField(null=True, upload_to='certification_documents/'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='created_by_id',
            field=models.IntegerField(default=0),
        ),
        migrations.AddField(
            model_name='customuser',
            name='image',
            field=models.FileField(null=True, upload_to='profile/'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='is_delete',
            field=models.IntegerField(default=0),
        ),
        migrations.AddField(
            model_name='customuser',
            name='is_suspend',
            field=models.IntegerField(default=0),
        ),
        migrations.AddField(
            model_name='customuser',
            name='mode_of_payment_id',
            field=models.IntegerField(default=0, help_text='0=>online, 1=>Offline , 2> By Accrediated Member'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='payment_status_id',
            field=models.IntegerField(default=0, help_text='2=>Pending, 1=>Done, 0=> Cancel'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='phone_number',
            field=models.BigIntegerField(null=True),
        ),
        migrations.AddField(
            model_name='customuser',
            name='store_exception',
            field=models.CharField(max_length=500, null=True),
        ),
    ]
